### PR TITLE
Move docker add kubernetes

### PR DESCRIPTION
**Pull request by devclinton**
* Move docker related items to the docker sub-folder and add some instructions for use
* Setup a local version of kubernetes for running locally as well with instructions

### CODE REVIEW

**Comment by devclinton** [comment]
> Looks good, only question, do we need both docker and platform folders? Could we just have one merged one?

We probably could. Would you prefer that? Easy change to make

**Comment by devclinton** [comment]
> We may need to update this README too, otherwise LGTM : https://github.com/InstituteforDiseaseModeling/covasim/tree/master/covasim/webapp#docker

Good catch. I will make update shortly.

**Comment by devclinton** [comment]
> One folder could work too :) You could just put the docker into platform. I'd also consider naming it `.platform` so it stays at the top and out of the way (that's why `.github` is `.github` )

Will do

**Comment by devclinton** [comment]
I made the requested updates

**Comment by devclinton** [comment]
Resolved white space issues as well

File: .platform/covasim-service.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
```
Comment (inc0) [inline .platform/covasim-service.yaml:2]: Unless we are running 1 node k8s, in which case we shouldn't run any k8s, this should be `Deployment` with several replicas

Comment (devclinton) [reply .platform/covasim-service.yaml:2]: This isn't the production version. I intended this for local testing. I will include the true production version as well which includes replicas. Should we have one for local testing then or just one for for both environments?

Comment (inc0) [reply .platform/covasim-service.yaml:2]: I'd run just one. You can run deployments on microk8s/minikube no problem, even with replicas.

Comment (devclinton) [reply .platform/covasim-service.yaml]: I add replicas and also ingress files for azure and microk8s. @inc0 Please review. My kubernetes knowledge is probably not as deep as yours

**Pull request merged**